tests/integration/defs/accuracy/test_llm_api_pytorch.py

```
@@ -751,6 +751,24 @@ def test_auto_dtype(self):
             task = GSM8K(self.MODEL_NAME)
             task.evaluate(llm)
 
+    def test_fp8_prequantized(self):
+        # Disabling kv cache reuse as a WAR to deal with gaps in kernel support for Gemma3's non-inclusive sliding window size.
```

Ruff (0.12.2)

Line too long (127 > 120)

(E501)